virttest/qemu_vm.py

```diff
@@ -1517,9 +1513,9 @@ def sort_key(dev):
                 pcics.sort(key=sort_key, reverse=False)
             devices.insert(pcics)
 
-        def add_secure_guest_object(params):
+        def add_secure_guest(params):
```

Comment:
I would suggest adding some suffixes, `add_secure_guest` make me so confused, like adding a real guest.

Reply:
Yes, make sense, have updated the method name to `add_secure_guest_descriptor`.